packages/eas-cli/src/build/utils/printBuildInfo.ts

```diff
@@ -15,6 +15,25 @@ import Log, { learnMore, link } from '../../log';
 import { appPlatformDisplayNames, appPlatformEmojis } from '../../platform';
 import { getBuildLogsUrl, getInternalDistributionInstallUrl } from './url';
 
+const errorCodeToErrorMessageOverride: Record<string, (build: BuildFragment) => string> = {
+  EAS_BUILD_UNKNOWN_FASTLANE_ERROR: build =>
+    `The ${link(getBuildLogsUrl(build, 'run-fastlane'), {
+      text: '"Run fastlane"',
+    })} step failed with an unknown error. Refer to the ${link(
+      getBuildLogsUrl(build, 'xcode-logs'),
+      {
+        text: '"Xcode logs"',
```

Comment:
depending on the terminal emulator it will either print "Xcode logs" or full url. This message should make sense in both cases.
Alternatively you can check if terminallink is supported and prepare to different messages

Reply:
It will print either clickable `"Xcode logs"` or `"Xcode logs": URL` if terminallink is not available. I think this is correct behavior, I already tested it.

What do you think?

Reply:
Then this sentence will go "Refer to the "Xcode logs": https://long.url.com  phase" that does not look right, with a random colon inside the sentence, plus it's not clear what the word phase refers to there. I would suggest just writing 2 versions, one with supported terminals and one for unsupported.

Reply:
Thanks for the feedback! It makes sense 👍

Reply:
<img width="635" alt="Screenshot 2023-02-16 at 12 14 11" src="https://user-images.githubusercontent.com/55145344/219350334-0e6881ac-2387-4e84-afec-3a708b1b17bb.png">
What do you think?

Reply:
CC: @brentvatne

Reply:
lgtm!